driver/src/main/scala/com/stratio/crossdata/driver/Driver.scala

```diff
@@ -129,7 +135,9 @@ class Driver(properties: java.util.Map[String, ConfigValue], flattenTables: Bool
     * @param retries Number of retries if the timeout was exceeded
     * @return A list of rows with the result of the query
     */
-  def asyncQuery(sqlCommand: SQLCommand, timeout: Timeout = Timeout(10 seconds), retries: Int = 2): Future[SQLResult] = {
+  def asyncQuery(sqlCommand: SQLCommand,
+                 timeout: Timeout = Timeout(Driver.config.getDuration(DriverRetryDuration, MILLISECONDS), MILLISECONDS),
+                 retries: Int = Driver.config.getInt(DriverRetryTimes)): Future[SQLResult] = {
```

Comment:
Instead of calculating the values of timeout and retries every time that asyncQuery and syncQuery are invoked, I'd calculate these values once in a lazy mode in the object part.